### PR TITLE
storage: Merge Host and HostCreds models into same etcd key.

### PR DESCRIPTION
**Pull request by mbarnes**
This is an ugly but temporary hack to keep E2E tests working until the Custodia integration is finished.  I'll probably revert this commit in the next week or two, but we can't leave code in "master" broken.

Re-run tests after https://github.com/projectatomic/commissaire-http/pull/80 is merged.

### CODE REVIEW

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
```
src/commissaire/storage/etcd.py:141:13: E129 visually indented line with same indent as next logical line
```

File: src/commissaire/storage/etcd.py [hunk 1]
```diff
@@ -131,15 +134,30 @@ def _save(self, model_instance):
...
-        model_instance = model_type.new(**model_dict)
-        if etcd_resp.newKey:
-            self.notify.created(model_instance)
-        else:
-            self.notify.updated(model_instance)
+        json_string = model_instance.to_json()
+        # XXX Temporary hack to merge Host and HostCreds models
+        #     to the same etcd key.
+        if (isinstance(model_instance, models.HostCreds) or
+            isinstance(model_instance, models.Host)):
```
Comment (ashcrow) [inline src/commissaire/storage/etcd.py]: This line needs 4 more spaces.

**Comment by mbarnes** [comment]
@ashcrow  :arrow_up:

**Review by ashcrow — APPROVED** [review]

**Comment by ashcrow** [comment]
Merged manually.

**Pull request closed without merging**